### ISSUE TITLE
Error: unable to load R code in package ‘R6’ / unexpected end of input

### ISSUE THREAD

**Issue by maek-ies** [posit-dev/r-shinylive#68]
The following error started to occur for me today when trying shinyapps at https://shinylive.io/r/examples/ 

I tried "empty cache and hard reload", manual clearing of the cache from browser settings, incognito mode and an alternative browser (Edge) but the error still occurs and shiny app never loads. Maybe related to the recent webR 0.3.1 version release?

![2024-03-25_10-25-19](https://github.com/posit-dev/r-shinylive/assets/151060752/62f534e6-7ce7-4faa-a4fa-1934a328d1e2)

![image](https://github.com/posit-dev/r-shinylive/assets/151060752/28993f43-34e1-4416-8b87-91292a93f572)

**Comment by gadenbuie** [comment]
Hi @maek-ies, thanks for the report. We're working to fix a small issue with the package repository used by shinylive. It should be back in working order in an hour or so.

**Comment by maek-ies** [comment]
Dear Garrick, thank you for letting me know. 

Good to hear that you are working on the issue. On my side, the error changed since the morning and is now similar to that experienced by @seanbirchall in #67. Except it now tries to mount Rccp twice.

```r
`Unable to mount image, /usr/lib/R/library/Rcpp is already mounted.
code for methods in class “RawReader” was not checked for suspicious field assignments (recommended package ‘codetools’ not available?)
Error: there is no package called jsonlite
Error: there is no package called renv
Error: object app_gu3mrbi39akaclw36q3m not found
Error: there is no package called shiny`
```

**Comment by gadenbuie** [comment]
@maek-ies this should be all cleared up now. You may need to clear your browser cache for shinylive.io. (In Chrome, under **View**, click **Clear cache and refresh**.)

**Comment by maek-ies** [comment]
Dear @gadenbuie, many thanks for solving the issue so quickly. On my side, the issue with attempting to mount Rcpp again still persists, even after clearing the cache (I also cleared all cache manually in the browser). But I think it might be related to the caching at proxy level in my organization because on the personal computer the issue is fixed (and it present when I checked yestereday). 

![2024-03-26_10-47-55](https://github.com/posit-dev/r-shinylive/assets/151060752/15f4e708-631f-4da6-95a3-c95cb5a25d84)
